### PR TITLE
Hashing

### PR DESCRIPTION
**Pull request by BairamovAllaz**

### CODE REVIEW

**Comment by BairamovAllaz** [comment]
Fixed #47

File: routes/user.js [hunk 1]
```diff
@@ -48,11 +48,16 @@ router.post(
...
+    const salt = await bcrypt.genSalt();
+    
+    const passwordHash = await bcrypt.hash(req.body.password,salt);
+
     const user = {
       first_name: req.body.first_name,
       last_name: req.body.last_name,
       email: req.body.email,
-      hashedPassword: req.body.password,
+      hashedPassword: passwordHash,
```
Comment (DamyanBG) [inline routes/user.js:60]: I would want to change this hashedPassword key too.

**Review by DamyanBG — COMMENTED** [review]
On PUT request we have some logic also. And it is good on all GET requests related to users to remove the password from the response.

**Comment by DamyanBG** [comment]
I am closing this PR

**Pull request closed without merging**